charts/argo-workflows/values.yaml

```diff
@@ -70,6 +70,8 @@ controller:
   rbac:
     # -- Adds Role and RoleBinding for the controller.
     create: true
+    # -- Allows controller to get, list, abd watch certain k8s secrets
+    secretWhitelist: {}
```

Comment:
`secretWhitelist` is supposed to be list( `[]` ), I think. :eyes:

Reply:
both are ok, I tested it with `--set "controller.rbac.secretWhitelist={secret0,secret1}"` and the clusterrole looked like below. I changed it to `[]` because it looks better :)
```
- apiGroups:
  - ""
  resourceNames:
  - secret0
  - secret1
  resources:
  - secrets
  verbs:
  - get
  - list
  - watch
```

Reply:
Thank you for changing! 
*We just need list of string in this case in stead of object, so list is better in the aspect of type.